core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java

```diff
@@ -212,7 +212,7 @@ public void afterRestore(Context<? extends Resource> context) throws Exception {
     private static Application manualApp;
     private static final int MANUAL_BEGIN = 0;
     private static final int MANUAL_BEGIN_INITIALIZATION = 1;
-    private static final int MANUAL_INITALIZED = 2;
+    private static final int MANUAL_INITIALIZED = 2;
```

Comment:
what does this change have to do with syncing docs?

Reply:
I think it was a stray rebase commit. Fixed/removed.